### PR TITLE
Create index.ts

### PR DESCRIPTION
**Pull request by Rcxz0**

### CODE REVIEW

**Comment by MrDiamondDog** [comment]
LGTM

**Comment by Luna-devv** [comment]
+1

**Review by ImLvna — APPROVED** [review]
LGTM

File: src/userplugins/epicPlugin/index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import definePlugin from "@utils/types";
```
Comment (TheKodeToad) [inline src/userplugins/epicPlugin/index.ts:1]: Missing license header

File: src/userplugins/epicPlugin/index.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export default definePlugin({
+    name: "Plugin test",
+    description: "Plugin test",
+    authors: [
+        {
+            id: 570717618323980321n,
+            name: "Riham",
+        },
```
Comment (TheKodeToad) [inline src/userplugins/epicPlugin/index.ts:10]: Should be in Devs

File: src/userplugins/epicPlugin/index.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    ],
+    patches: [],
+    // Delete these two below if you are only using code patches
+    start() {},
+    stop() {},
```
Comment (TheKodeToad) [inline src/userplugins/epicPlugin/index.ts:15]:
```suggestion
```

**Review by TheKodeToad — CHANGES_REQUESTED** [review]

**Comment by ImLvna** [comment]
@TheKodeToad its a userplugin it doesnt follow any of those rules 🙄🙄🙄

**Pull request closed without merging**